ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -1148,13 +1154,18 @@ const ControlledTierCard = (props: {
             spinnerProps={spinnerProps}
           />
         )}
-        <ModulesForInitialRegistration
-          profile={profile}
-          modules={[ctModule]}
-          activeModule={activeModule}
-          clickableModules={clickableModules}
-          spinnerProps={spinnerProps}
-        />
+        {pageMode === DARPageMode.INITIAL_REGISTRATION && (
+          <ModulesForInitialRegistration
+            profile={profile}
+            modules={[ctModule]}
+            activeModule={activeModule}
+            clickableModules={clickableModules}
+            spinnerProps={spinnerProps}
+          />
+        )}
+        {pageMode === DARPageMode.ANNUAL_RENEWAL && isEligible && (
```

Comment:
<img width="970" alt="image" src="https://user-images.githubusercontent.com/24514630/163192923-096bdac9-9d4a-4d25-acac-7ddfa2728ff9.png">
 Good call @jmthibault79 on hiding the training when not eligible.